### DISCUSSION TITLE
Thoughts on the new bookmarking feature

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#4454] [General]
I think the new bookmarking by hostname feature is really cool, its great for servers to not have to worry as much about their IP changing. I'm a little late to this i guess, didn't really get to mess with it on preview, but to me it seems strange to have two separate 'favorite' features, I had thought bookmarking would be a replacement to favorites more than along-side it. Have you considered making favorites show up in the bookmarks menu too, or just allowing IPs to be used in bookmarks along with hostnames?

**Comment by SDGNelson** [comment]
Thanks! Ideally long-term bookmarks will supersede favorites - especially if we add server history tracking on top of it - but considering the controversy of the server list menu changes I figure best to play it safe for the meantime. It'll also take some time for hosts to become familiar and start setting BookmarkHost since it doesn't show up unless set.

If you're confident the IP will stay the same you can set BookmarkHost to your public IP! But then if the IP does change it won't be able to update the bookmark until the player joins using the new IP.

**Reply by DanielWillett** [reply]
Makes sense, thanks.

> If you're confident the IP will stay the same you can set BookmarkHost to your public IP! 

didnt think about this actually
